### ISSUE TITLE
Export items in the Queue

### ISSUE THREAD

**Issue by tanimasarkhel** [taskforcesh/issues#34]
Is it possible for a taskforce user, to export the details of all the jobs that has been processed in the queue in an excel file from Taskforce?

**Comment by manast** [comment]
No, that is not possible, however using Bull apis you should be able to get the jobs within a giving range and convert the data to CSV format that can then be imported to excel.

**Comment by swhiteman** [comment]
Re-upping this. I have no need for Excel, a JSON export would fit a lot better. But after I run a bunch of jobs it would be great to snapshot the Failed and Completed tabs into separate files.

(Not that I can’t generate something like this using the BullMQ API, as you say. But seems imo like a purpose-built monitoring tool like Taskforce is going to be way better at exporting its state. Otherwise I’m writing duplicative code and probably screwing up somehow.)

**Comment by manast** [comment]
Ok, we will look into the possibility to add this improvement asap.
